install/kubernetes/helm/istio/charts/kiali/templates/secrets.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kiali
+  labels:
+    app: kiali
+
+type: Opaque
+data:
+  username: {{ .Values.dashboard.username }}
+  password: {{ .Values.dashboard.password }}
```

Comment:
I am not sure if this is going to be correct. These secrets are "Opaque" so they have to be encrypted. I am assuming the .Values.dashboard.X values are in cleartext. We need these encrypted - as you see here: https://github.com/kiali/kiali/blob/master/deploy/kubernetes/kiali-secrets.yaml#L10-L11

Reply:
Thanks, @jmazzitelli for pointing that out.
I think 84ad47d may solve it.